index.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const server = express();
+const cors = require('cors');
+const PORT = process.env.port || 5000;
+const helmet = require('helmet');
+const morgan = require('morgan');
+const authRouter = require('./routers/authRouter')
+const postRouter = require('./routers/postRouter');
+require('dotenv').config()
+
```

Comment:
Well structured server file Michael 👍 , great to see that you've modularized your code with routers too 🔥